ReleaseProcessAutomation/Configuration/ConfigReader.cs

```diff
@@ -59,9 +59,26 @@ public Config LoadConfig (string configPath)
       throw new InvalidOperationException(message);
     }
 
-    config.MSBuildSettings.MSBuildPath = Path.GetFullPath(config.MSBuildSettings.MSBuildPath);
+    var fullPath = Path.GetFullPath(config.MSBuildSettings.MSBuildPath);
+    if (File.Exists(fullPath))
+    {
+      config.MSBuildSettings.MSBuildPath = fullPath;
+      return config;
+    }
+      
 
-    return config;
+    foreach (var alternativeMSBuildPath in config.MSBuildSettings.AlternativeMSBuildPath)
+    {
+      fullPath = Path.GetFullPath(alternativeMSBuildPath);
+      if (!File.Exists(fullPath))
+        continue;
+
+      _log.Information("Using MSBuild from the path '{FullPath}'.", fullPath);
+      config.MSBuildSettings.MSBuildPath = fullPath;
+      return config;
+      
+    }
+    throw new ArgumentException("None of the configured MSBuild paths in the config exist.\nPlease configure a proper MSBuild path in the config.");
```

Comment:
We might be able to simplify this code into a single LINQ query that returns the first path that exists or null. I think this would improve the readability here.